app/main/views/briefs.py

```diff
@@ -226,6 +226,10 @@ def redirect_to_next_page():
                 return redirect(url_for('.view_response_result', brief_id=brief_id))
 
     previous_question_id = section.get_previous_question_id(question_id)
+    # Skip previous question if the brief has no nice to have requirements
+    if previous_question_id in brief.keys() and not brief[previous_question_id]:
```

Comment:
Yep, have thought about it further and believe what currently stands is correct. Nice to have requirements is attributed to the brief and will come falsey if there are no nice to haves. Questions such as `availability` does not exist on the brief so would return `False` to `if not brief.get(previous_question_id)` and skip the question, which we do not want to do as they should not be skipped.

There might be more elegant ways to do this check (or the blunt way of hardcoding `niceToHaveRequirements` and deciding based on that) but I think I'll merge it as it stands.